config/routes.rb

```diff
@@ -199,7 +199,7 @@
   get 'projects/:project_id/wiki/:id/toc' => 'wiki#index', as: 'wiki_page_toc'
   post 'projects/:project_id/wiki/preview' => 'wiki#preview', as: 'preview_wiki'
   post 'projects/:id/wiki' => 'wikis#edit'
-  match 'projects/:id/wiki/destroy' => 'wikis#destroy'
+  delete 'projects/:id/wiki/destroy' => 'wikis#destroy'
```

Comment:
With this in mind, you can probably remove the `verify`at https://github.com/opf/openproject/blob/feature/rails4/app/controllers/wiki_controller.rb#L278.

And while you are at it, there are other `verify` statements in that controller as well.